Tuebingen/UBTaeth2.xml

```diff
@@ -0,0 +1,273 @@
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ ዘበዓለ፡ ፵፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.14">
+  <locus from="145v" to="158"/>
+  <title type="complete" ref="LIT2137OntheP"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ ዘበዓለ፡ ፶፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.15">
+  <locus from="158v" to="163r"/>
+  <title type="complete" ref="LIT6047On12ApostlesRetuaH"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፭ለሐምሌ፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.16">
+  <locus from="163r" to="169v"/>
+  <title type="complete" ref="LIT2145OntheT"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፲ወ፫፡ ለነሐሴ።</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.17">
+  <locus from="170r" to="189r"/>
+  <title type="complete" ref="LIT1640Homily"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፲፮፡ ለነሐሴ።</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.18">
+  <locus from="189v" to="196v"/>
+  <title type="complete" ref="LIT2124OntheF"/>
+  <note>Indication for reading: <foreign xml:lang="gez">ምንባብ፡ አመ፡ ፰፡ ለኅዳር፡</foreign></note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+<colophon xml:id="coloph1" xml:lang="gez">
+<locus target="#197r"/>
+ተፈጸመ፡ መጽሐፈ፡ ርቱዓ፡ ሃይማኖት፡ በወርኃ፡ <sic>ማዝያ፡ </sic> በዘመነ፡ ዮሐንስ፡ በ፲ወ፯፡ ዕለት፡ ወሌሊት፨ በዕለተ፡ በዓለ፡ ስቅለቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ዘተወጥነ፡
+በወርኃ፡ ጥር፡ በ፲ወ፪ዕለት፡ ወበ፲ሌሊት፨ በሀገረ፡ <placeName ref="LOC1463Ankoba">አንኮበር</placeName>፡ በብሔረ፡ ሸዋ፡
+</colophon>
+
+  </msItem>
+</msContents>
+
+<physDesc>
+
+  <objectDesc form="Book">
+      <supportDesc>
+          <support>
+              <material key="paper"/>
+          </support>
+          <extent>
+              <measure unit="page">197</measure>
+              <dimensions type="outer" unit="mm">
+                  <height>245</height>
+                  <width>190</width>
+              </dimensions>
+          </extent>
+        </supportDesc>
+
+      <layoutDesc>
+          <layout columns="2" writtenLines="23 26" cert="high">
+          <locus from="1r" to="141v"></locus>
+          <locus from="154r" to="159v"></locus>
+          </layout>
+
+          <layout columns="1" writtenLines="34" cert="high">
+          <locus from="142r" to="153v"></locus>
+          <locus from="160r" to="197r"></locus>
+          </layout>
+
+      </layoutDesc>
+  </objectDesc>
+  <handDesc>
+    <handNote script="Ethiopic" xml:id="h1">
+          <desc>Written in a small, regular hand, of the <date notBefore="1800" notAfter="1900">19th Century</date>.</desc>
+    </handNote>
+  </handDesc>
+
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<locus target="#106v #125r"/>
+<desc type="ScribalSupplication">The scribal supplication at the end of some homilies mentions the scribe
```

Comment:
Since this is written within the main text, I would go for e instead of a

Reply:
Done!